### PR TITLE
Ui restyle of Health box in HealthSummary

### PR DESCRIPTION
**Pull request by CDimonaco**
# Description

As title.

We add a smart component for the health boxes with the new UI style and the ability to acts as button with a 'selected" state, like a checkbox. This is done for preparing the boxes health filtering feature.

Tag @jagabomb for ui review

image

The style of the boxes changes in all the views. 
## How was this tested?

Manually.

### CODE REVIEW

**Comment by abravosuse** [comment]
Code apart, looks good to me.

File: assets/js/components/HealthSummary/HealthSummaryBox.jsx [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+import classNames from 'classnames';
+import React from 'react';
+
+const iconByHealth = {
+  passing: () => (
+    <EOS_CHECK_CIRCLE_OUTLINED size={40} className="fill-green-600" />
+  ),
+  warning: () => <EOS_WARNING_OUTLINED size={40} className="fill-yellow-400" />,
+  critical: () => <EOS_ERROR_OUTLINED size={40} className="fill-red-600" />,
+};
```
Comment (dottorblaster) [inline assets/js/components/HealthSummary/HealthSummaryBox.jsx:15]: Maybe just a simple switch statement could have worked here?

**Review by dottorblaster — APPROVED** [review]
LGTM

**Review by jagabomb — APPROVED** [review]
LGTM

File: assets/js/components/HealthSummary/HealthSummaryBox.jsx (hunk 1 above)
Comment (CDimonaco) [reply assets/js/components/HealthSummary/HealthSummaryBox.jsx:15]: Honestly I think the map approach is more readable, having a switch case returning a function that later I use to render that component in another component 👀 , it's not my cup of tea

**Pull request merged**